dev/com.ibm.ws.com.lightbend.microprofile.reactive.streams.zerodep/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>com.ibm.ws.com.lightbend.microprofile.reactive.streams.zerodep</name>
```

Comment:
Are you sure we can't call this just `com.ibm.ws.com.lightbend.zerodep`?

Reply:
I have followed the convention - so personally I think it is more 'engineering' not to create shortenings/psuedonymns beyond the convention if noone has to type this name in.